django/test/runner.py

```diff
@@ -1198,8 +1198,8 @@ def reorder_tests(tests, classes, reverse=False, shuffler=None):
         yield from reorder_test_bin(tests, shuffler=shuffler, reverse=reverse)
 
 
-def partition_suite_by_case(suite):
-    """Partition a test suite by test case, preserving the order of tests."""
+def partition_suite_by_test_case_class(suite):
```

Comment:
This PR should be limited to docs changes  (possibly comments/docstrings). It's not worth changing function names.